cmip6_downscaling/workflows/bcsd_flow.py

```diff
@@ -14,9 +14,13 @@
     return_gcm_train_full_time,
     return_obs,
 )
+from cmip6_downscaling.tasks import pyramid
 
 runtime = runtimes.get_runtime()
 
+config.set({'storage.intermediate.uri': 'az://flow-outputs/intermediates'})
+config.set({'storage.results.uri': 'az://flow-outputs/results'})
+config.set({'storage.temporary.uri': 'az://flow-outputs/temporary'})
```

Comment:
I think this should work like this:
```suggestion
config.set(
    {
        'storage.intermediate.uri': 'az://flow-outputs/intermediates',
        'storage.results.uri': 'az://flow-outputs/results',
        'storage.temporary.uri': 'az://flow-outputs/temporary'
    }
)
```

But, before we merge this, I think we should move this sort of configuration to your `yaml` config file: https://donfig.readthedocs.io/en/latest/configuration.html#yaml-files

As a general rule, I think we should avoid setting these values in the flow module.